pwiz_tools/Skyline/TestTutorial/CustomReportsTutorialTest.cs

```diff
@@ -494,14 +495,23 @@ protected bool DoQualityControlSummaryReports()
             RunUI(SkylineWindow.ShowPeakAreaReplicateComparison);
             WaitForGraphs();
 
-            PauseForPeakAreaGraphScreenShot("Peak Areas view");
+            PauseForScreenShot<GraphSummary.AreaGraphView>("Peak Areas View");

```

Comment:
Though, the class name is AreaGraphView, we generally do not capitalize "View" in referring to our views. The name "Peak Areas" gets capitalized because it is a "proper name" that appears in the UI. But the term "view" does not appear in the UI, it is just something we use to qualify the proper name.

Reply:
Typo, my bad - good catch.